ui/src/groups/GroupAdmin/GroupInfoEditor.tsx

```diff
@@ -67,8 +68,11 @@ export default function GroupInfoEditor({ title }: ViewProps) {
   }, [groupFlag, navigate]);
 
   const onSubmit = useCallback(
-    (values: GroupMeta) => {
-      useGroupState.getState().edit(groupFlag, values);
+    async (values: GroupMeta & { privacy: PrivacyType }) => {
+      await useGroupState.getState().edit(groupFlag, values);
```

Comment:
This doesn't full address privacy setting, it needs to also set the cordon appropriately like we do in the NewGroup form